README.md

```diff
@@ -215,7 +215,7 @@ marathon-protocol           | `http`          | Marathon protocol (http or https
 marathon-ssl-verify         | `true`          | Verify certificates when connecting via SSL
 marathon-timeout            | `30s`           | Time limit for requests made by the Marathon HTTP client. A Timeout of zero means no timeout
 marathon-username           |                 | Marathon username for basic auth
-marathon-leader             |                 | Marathon cluster-wide node name (defaults to <hostname>:8080), the some leader specific calls will be made only if the specified node is the current Marathon-leader")
+marathon-leader             | `<hostname>:8080` | Marathon cluster-wide node name, the some leader specific calls will be made only if the specified node is the current Marathon-leader. Set to `*` to always act like a Leader.
```

Comment:
This part of readme is copied from [`config/config.go`](https://github.com/allegro/marathon-consul/blob/master/config/config.go) could you update description there too? We should make it autogenerated.